Client/Frontend/Browser/SearchEngines/OpenSearchEngine.swift

```diff
@@ -93,8 +93,6 @@ class OpenSearchEngine: NSObject, NSSecureCoding {
            let fromTemplateURL = getURLFromTemplate(searchTemplate, query: query),
            fromTemplateURL.baseDomain != "ecosia.org" {
             return fromTemplateURL
-        } else if BingDistributionExperiment.isEnabled {
-            return BingDistributionExperiment.searchURLForQuery(query)
```

Comment:
Given `BingDistributionExperiment` still exists, I'd also argue this is needed if it's enable given this is the main logic for the experiment.